### ISSUE TITLE
"Symbolic link privilege not held" OSError when trying to add new videos to project

### ISSUE THREAD

**Issue by natalier11** [DeepLabCut/DeepLabCut#1199]
Hi all!

I'm still fairly new to using DLC (and coding terminals in general) but I've recently run into a small issue. I am trying to analyze a total of 95 5-10 minute long videos. I figured I would randomly select 10 of them, extract 20 frames from each, and label these 200 frames to then apply to the rest. I was able to extract frames and label the first ten videos but ran into trouble when trying to add the other 85 videos prior to creating the training dataset. I loaded the project in the GUI by selecting the config file with no problem and could press the "Load New Videos" button to select he other 85 videos. However, when I press "Add New Videos," I receive the following error:
![DLC symbolic link privilege error](https://user-images.githubusercontent.com/73397911/115590416-8b40b580-a296-11eb-9040-8645402c5527.png)
I've tried this on two separate computers (one is a personal computer with no admin blocks and one is an admin run system) and I have had the same issue both times. I was wondering if there was something I'm doing incorrectly or if there's a way to mass add the videos through the terminal that I'm missing. I'd also love to know of there's an easier way to only extract frames from and label a portion of the videos besides this or using the manual "need user feedback" button on the extract frames tab of the GUI (kind of a pain to type yes or no 95 times). Thanks all!

**Comment by backyardbiomech** [comment]
You don't need to add the videos that aren't part of training.
Adding videos prior to training is ONLY for including them in training (i.e. your labeled 10). Once the model is trained, you can apply the model to other videos in the Analyze Videos tab.

When you get to analysis, if all of the videos are in the same folder, you can select just that folder and then select the correct movie format, and DLC will go analyze all of the videos with that extension in the selected folder.

After analyzing, if you want to extract outliers and refine from a video that wasn't included in the original training set, you will run into this symbolic link privilege not held error. The easiest way to avoid that is to open the Anaconda command prompt as an administrator.

**Comment by natalier11** [comment]
Thanks so much for letting me know! Is this something I can do outside of the GUI and if so, how? I've been using an online supercomputer called Cheaha and a jupyter notebook to do all the computing-heavy work (creating the training dataset and everything after that) and using the following line of code for the analysis:
`deeplabcut.analyze_videos(path_config_file,videofile_path, videotype=VideoType)`
with the videofile_path defined as such:
`videofile_path = ['/data/user/nrem/deeplabcut/'+ProjectFolderName+'/videos/']`
Point is, do I have to have the videos I want to analyze in the project folder or can they come from elsewhere?

**Comment by backyardbiomech** [comment]
Videos you are analyzing can be outside of the project folder. That is what the `videofile_path` variable is for. You can change that to any folder containing all your videos or a list of individual files anywhere.

You can see details and some examples on the docs pages:
[https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/functionDetails.md](https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/functionDetails.md)
[https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/UseOverviewGuide.md](https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/UseOverviewGuide.md)
[https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/HelperFunctions.md](https://github.com/DeepLabCut/DeepLabCut/blob/master/docs/HelperFunctions.md)

**Comment by MMathisLab** [comment]
Note, the error is because you need to be sure to right-click "open as admin" on the conda prompt (even if you are an admin on the system). and +1 to everything else @backyardbiomech said!

**Comment by natalier11** [comment]
Thanks you guys!! This was super helpful and I think I've got it working now, woohoo!!!
